react/src/lib/components/SubsurfaceViewer/layers/polylines/polylinesLayer.ts

```diff
@@ -8,6 +8,9 @@ import {
     defineBoundingBox,
 } from "../utils/layerTools";
 
+type TIsPolylineClosedFunc = (index: number) => boolean;
```

Comment:
Not a huge fan of the "T" prefix. I don't think it helps readability and you can usually see that it's a type in the editor.

Reply:
T prefixes deleted